docs/useful_tools.md

```diff
@@ -138,13 +138,44 @@ If you need a lightweight GUI for visualizing the detection results, you can ref
 ## Error Analysis
 
 `tools/analysis_tools/coco_error_analysis.py` analyzes COCO results per category and by
- different criterion. It can also make a plot to provide useful
-  information.
+ different criterion. It can also make a plot to provide useful information.
 
 ```shell
 python tools/analysis_tools/coco_error_analysis.py ${RESULT} ${OUT_DIR} [-h] [--ann ${ANN}] [--types ${TYPES[TYPES...]}]
 ```
 
+Example:
+
+Assume that you have got Mask R-CNN checkpoint file in the path 'checkpoint'. You can use the following command to get the results bbox and segmentation json file
```

Comment:
1. json file -> json file.
2. Provide guide to download the model in case the users do not know where to find a model.